### PR TITLE
[bounty] Add "Informe del responsable de mantenimiento de BCHN 15-04-2020" article

### PR DESCRIPTION
**Pull request by merc1er**
#66

### CODE REVIEW

**Review by ftrader — CHANGES_REQUESTED** [review]
Only one change requested (include the translator reference at top)

File: blog/Informe-del-responsable-de-mantenimiento-de-BCHN-15-04-2020.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
+---
+layout: layout.html
+---
+
+<% set('title', 'Informe del responsable de mantenimiento de BCHN 15-04-2020') %>
+<% set('date', '29 April 2020') %>
+<% set('author', 'freetrader') %>
+
+
```
Comment (ftrader) [inline blog/Informe-del-responsable-de-mantenimiento-de-BCHN-15-04-2020.md:9]: Should include

> (Esta es una traducción realizada por [Grido](https://read.cash/@Grido) del artículo [original](https://bitcoincashnode.org/en/newsroom/BCHN-lead-maintainer-report-2020-04-15) en inglés.)

**Comment by merc1er** [comment]
> Only one change requested (include the translator reference at top)

Updated

**Review by ftrader — APPROVED** [review]

**Comment by ftrader** [comment]
LGTM, bounty awarded to @merc1er

**Pull request merged**